cmd/node/config/config.toml

```diff
@@ -588,6 +588,9 @@
         SameSourceRequests = 10000
         # SameSourceResetIntervalInSec time frame between counter reset, in seconds
         SameSourceResetIntervalInSec = 1
+        # MaxGasPerVmQuery defines the maximum amount of gas to be allocated for VM Queries comming from API
+        # If set to 0, then MaxUInt64 will be used
+        MaxGasPerVmQuery = 1500000000  #1.5b
```

Comment:
This has nothing to do with the antiflood param. Please move it in the vm queries configs from api

Reply:
moved